internal/controller/authenticator/jwtcachefiller/jwtcachefiller_test.go

```diff
@@ -87,6 +90,59 @@ func TestController(t *testing.T) {
 		}
 		require.NoError(t, json.NewEncoder(w).Encode(jwks))
 	}))
+	// Claims without the subject, to be used distributed claims tests.
+	// OIDC 1.0 section 5.6.2:
+	// A sub (subject) Claim SHOULD NOT be returned from the Claims Provider unless its value
+	// is an identifier for the End-User at the Claims Provider (and not for the OpenID Provider or another party);
+	// this typically means that a sub Claim SHOULD NOT be provided.
+	claimsWithoutSubject := jwt.Claims{
+		Issuer:    server.URL,
+		Audience:  []string{goodAudience},
+		Expiry:    jwt.NewNumericDate(time.Now().Add(time.Hour)),
+		NotBefore: jwt.NewNumericDate(time.Now().Add(-time.Hour)),
+		IssuedAt:  jwt.NewNumericDate(time.Now().Add(-time.Hour)),
+	}
+	mux.Handle("/claim_source", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
This test feels pretty awkward but I can't declare `mux.Handle("/claim_source", ...` later (once the groups claim name and groups value are known) because it's not allowed to be redeclared for each test.
I thought about spinning up an entire new test server in each test but that also gets awkward because currently we need to know the issuer name in the test table and it also just feels kinda wasteful.